application-services/README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Overview
 
-This folder contains various examples of Application Services based on the App Functions SDK and Profiles that can be used with App Service Configurable. See the [Application Services](https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices/.) documentation for complete details on Application services.
+This folder contains various examples of Application Services based on the `Ireland 2.0.x` App Functions SDK and Profiles that can be used with App Service Configurable. See the [Application Services](https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices/.) documentation for complete details on Application services.
```

Comment:
Should we use `https://docs.edgexfoundry.org/2.0/` as the version for Docs?

Reply:
Good catch. Thought I had got all those. ;-)